graph-courses.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
```

Comment:
This file should be renamed, as it isn't graphing courses any more. Perhaps something like `import-courses.py` would be appropriate.